### PR TITLE
A locale in this context is a comination of a language, a script and …

### PR DESCRIPTION
**Pull request by michaelKaefer**
…a region

### CODE REVIEW

**Review by OskarStark — COMMENTED** [review]
friendly ping @ro0NL, I would love to get your review on this PR 🙃

**Comment by javiereguiluz** [comment]
What about using something similar to the [Wikipedia definition of "locale"](https://en.wikipedia.org/wiki/Locale_(computer_software)):

```
A locale is the combination of a language, a region and some parameters that
define the interface preferences of the user (such as the language script, the
date and number formatting, etc.).
```

**Comment by michaelKaefer** [comment]
@javiereguiluz I think the definition on Wikipedia is to general for what is meant here. A locale here is almost always a combination of a language code, a country code and a _script code_ and not the combination of a language code, a country code and _some parameters_.

File: components/intl.rst [hunk 1]
```diff
@@ -210,10 +210,11 @@ You may convert codes between two-letter alpha2 and three-letter alpha3 codes::
 Locales
 ~~~~~~~
 
-A locale is the combination of a language and a region. For example, "Chinese"
-is the language and ``zh_Hans_MO`` is the locale for "Chinese" (language) +
-"Simplified" (script) + "Macau SAR China" (region). The ``Locales`` class
-provides access to the name of all locales::
+A locale is the combination of a language, a region and some parameters that
+define the interface preferences of the user. For example, "Chinese"is the
```
Comment (javiereguiluz) [inline components/intl.rst]: Note to mergers: add a space here -> `"Chinese"is`

**Comment by javiereguiluz** [comment]
@michaelKaefer thanks for updating your contribution. I wasn't sure about this either ... but looking at Symfony Intl tests (https://github.com/symfony/intl/blob/master/Tests/Locale/LocaleTest.php) it looks like we support locales with attributes like script, currency formatting, etc.

File: components/intl.rst (hunk 1 above)
Comment (michaelKaefer) [reply components/intl.rst]: Thank you, fixed it.

**Comment by michaelKaefer** [comment]
@javiereguiluz Ok, thank you. But `Locales::getNames()` only returns combinations of languages, scripts and countries, or? No currencies, no number formats, etc. That is what my commit was targeting. But anyway I already changed it to _some parameters_.

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
@michaelKaefer nice work! ... and thanks for your patience during the review process

**Pull request merged**